### ISSUE TITLE
exposure time vs. distance

### ISSUE THREAD

**Issue by traut21** [corona-warn-app/cwa-documentation#446]
From the FAQs I learned that the risk of exposure is calculated from distance and time.

What are the details how this is computed?

Is there a chart of time vs. distance or could someone provide the algorithm behind?

I wonder what the minimum contact time is,
e.g. when you pass someone in the underground train.

**Comment by ndegendogo** [comment]
@traut21 and don't forget to take into account that both dimensions (distance and duration) are **not** a precise measurement but only a rough estimation.

Distance is estimated based on the signal strength (attenuation), but the signal depends also on how you carry your device (open in your hand / in your back pocket), and on reflexions at the environment.

Duration is estimated based on the number of beacons received. But reception is performed every 5 minutes for a few seconds. So a single beacon can mean anything between 1 second and 10 minutes. And two readings (5 minutes apart) can mean anything between 5 and 15 minutes.

**Comment by MikeMcC399** [comment]
The question about the relationship between distance and dB levels was raised also in https://github.com/corona-warn-app/cwa-documentation/issues/484.

The time element is also discussed in https://github.com/corona-warn-app/cwa-documentation/issues/485 together with a new diagram on https://github.com/corona-warn-app/cwa-documentation/blob/master/images/risk_calculation/risk_calculation_enf_v2_overview.pdf.

**Comment by Ein-Tim** [comment]
Related: https://github.com/corona-warn-app/cwa-website/issues/712

**Comment by heinezen** [comment]
The comment from https://github.com/corona-warn-app/cwa-documentation/issues/484#issuecomment-781382074 should also answer this question I hope.

> The translation of signal attenuation to distance is not described by an equation, i.e. there is no formula that directly calculates the distance from signal attenuation. Given examples of attentuation-distance pairs (such as 73 dB --> >8m) should be seen as guidance values. The attenuation ranges used by the app's calculation are based on empiric research that - as far as I understand - examined which attenuation intervals would indicate that devices are close enough for the transmission of the virus.
> * 73dB is very far away
> * 63-73 is far away
> * 55-63 is mid-range
> * <55 is very close

The question about minimum exposure time is answered in the FAQ: https://www.coronawarn.app/de/faq/#encounter_19_calc

----
Corona-Warn-App Open Source Team

**Comment by Ein-Tim** [comment]
Is this issue solved?

**Comment by traut21** [comment]
not really. Some input is too much. The most relevant source https://github.com/corona-warn-app/cwa-documentation/blob/master/images/risk_calculation/risk_calculation_enf_v2_overview.pdf does not load properly and the downloaded version requires more detailed knowledge.

I do not understand the concept yet. If it's > 79 dB, the t_att is multiplied with 0,0 and then multiplied with a TRV.  I do not understand the concept why you compute something by  multiplying it with zero, making it zero that way. I would not expect an "x 0.0", but an "@ 0.0 s". Thus a contact that close would be added immediately.

https://github.com/corona-warn-app/cwa-documentation/blob/master/cwa-risk-assessment.md#current-configuration is more helpful, although I cannot find all matches between its tables and the other PDF. One is from 2021-01-19, the other from 2021-03-17. I guess the examples would need an update for the current status

**Comment by MikeMcC399** [comment]
@traut21 
https://github.com/corona-warn-app/cwa-documentation/blob/master/cwa-risk-assessment.md is very out of date. See https://github.com/corona-warn-app/cwa-documentation/issues/453.

**Comment by MikeMcC399** [comment]
@traut21 

The documentation has been updated.

See https://github.com/corona-warn-app/cwa-documentation/blob/main/solution_architecture.md#mobile-applications for current information.

The document https://github.com/corona-warn-app/cwa-documentation/blob/main/cwa-risk-assessment.md is now deprecated.

**Comment by Ein-Tim** [comment]
@traut21 See https://github.com/corona-warn-app/cwa-documentation/issues/446#issuecomment-1027704944, does this answer your question? If not: @dsarkar please mirror this issue to JRIA.

**Comment by traut21** [comment]
I'd prefer a more simple presentation
